plugins/cc/cctac.mli

```diff
@@ -10,8 +10,8 @@
 
 open EConstr
 
-val congruence_tac : int -> constr list -> unit Proofview.tactic
+val congruence_tac : int option -> constr list option -> unit Proofview.tactic
 
-val simple_congruence_tac : int -> constr list -> unit Proofview.tactic
+val simple_congruence_tac : int option -> constr list option -> unit Proofview.tactic
```

Comment:
For the list argument the default is empty list which IMO is not a magic value, so the API should just take list

Reply:
That's true. I thought it might be good to make it option because that's how it will be in the exposed grammar but you right that's nothing the cctac API should be concerned about.